src/components/Provider.tsx

```diff
@@ -24,12 +24,12 @@ export interface ProviderProps<A extends Action = AnyAction, S = any> {
   children: ReactNode
 }
 
-function Provider<A extends Action = AnyAction>({
+function Provider<A extends Action = AnyAction, S = any>({
```

Comment:
:question: Does it make sense for this to be `unknown` instead?

Reply:
Definitely, but `ProviderProps` uses `any` as the default so I had to do the same here. If you wish I can change both of those defaults to `unknown`? That's a larger change and I wasn't sure if you would want that.

Reply:
9e09869307f829273d8a38a452d814a546b692b0